pimd/pim_bsm.c

```diff
@@ -712,8 +712,9 @@ static bool pim_bsm_send_intf(uint8_t *buf, int len, struct interface *ifp,
 		return false;
 	}
 
-	if (pim_msg_send(pim_ifp->pim_sock_fd, pim_ifp->primary_address,
-			 dst_addr, buf, len, ifp->name)) {
+	if (pim_msg_send(pim_ifp->pim_sock_fd,
```

Comment:
Dont do alignment changes here, otherwise when you will raise next commit you will face conflict because you will have the changes in the same line.